pkg/util/provider/machinecontroller/machine_test.go

```diff
@@ -253,6 +260,15 @@ var _ = Describe("machine", func() {
 					machineObjects = append(machineObjects, o)
 				}
 
+				machine1 := &v1alpha1.Machine{
+					ObjectMeta: *newObjectMeta(objMeta, 0),
+					Spec: v1alpha1.MachineSpec{
+						Class: *data.action,
+					},
+				}
+
+				machineObjects = append(machineObjects, machine1)
+
```

Comment:
I see this `machine1` object being appended in `machineObjects` list but I don't see it being used in any testcases?

Reply:
You are correct, Himanshu. It is a redundant entry. 

The idea behind creating the machine object was to check that the machineClass gets a finaliser only when there is atleast one machine using it. I have written a different test case in `MachineClass_test.go` now. Again, good catch 😉